modules/displayBooks.js

```diff
@@ -0,0 +1,34 @@
+const displayBooks = (booksList) => {
+  const books = document.querySelector('.books');
+  let booksHtml = '';
+  booksList.list.forEach((book) => {
+    booksHtml += `<li class="book-item">
+                            <div class="book-info">
+                                <p class="book-title">${book.title.charAt(0).toUpperCase() + book.title.slice(1)}</p>
+                                <p class="book-by">by</p>
+                                <p class="book-author">${book.author}</p>
+                            </div>
+                            <button id="${book.id}" class="remove-btn" type="button">Remove</button>
+                       </li>
+              `;
+  });
+  books.innerHTML = booksHtml;
+
+  // remove book
+  const removeBtns = document.querySelectorAll('.remove-btn');
+  removeBtns.forEach((btn) => {
+    btn.addEventListener('click', (e) => {
+      /* eslint-disable */
+            const element = btn.parentNode;
+            booksList.removeBook(parseInt(e.target.id));
+            element.remove();
+        });
```

Comment:
- [ ] Let's not disable linter rules, I know sometimes it can be frustrating, but linter rules are here to help us avoid bugs and keep our code clean and standardized across teammates. In this particular file, I found that you're not using correctly the parseInt function, which requires two arguments, the string to be converted and a number between 2 and 36 that specifies the base (Usually 10). There are also some indentations issues, but nothing you can't correct.

Remember that you can always search on google what's the reason a linter rule might be triggering, I can assure you most of the time they are in place for a reason and will help you be a better developer.

Reply:
Thank you Daniel for your support 
I changed them upon the changes required, you can check now